app/src/main/java/com/rockthevote/grommet/data/api/RockyService.java

```diff
@@ -7,17 +7,20 @@
 import com.rockthevote.grommet.data.api.model.RegistrationResponse;
 import com.rockthevote.grommet.data.db.model.RockyRequest;
 
+import okhttp3.RequestBody;
+import retrofit2.Response;
 import retrofit2.adapter.rxjava.Result;
 import retrofit2.http.Body;
 import retrofit2.http.GET;
 import retrofit2.http.POST;
 import retrofit2.http.Query;
 import rx.Observable;
+import rx.Single;
 
 public interface RockyService {
 
     @POST("voterregistrationrequest")
-    Observable<Result<RegistrationResponse>> register(@Body RockyRequest rockyRequestWrapper);
+    Single<Result<RegistrationResponse>> register(@Body RequestBody rockyRequestBody);
```

Comment:
interesting work around, we lose type safety and it might make testing this a bit less straight forward since we can just pass anything and get a valid response,  But since this version of Grommet will be short lived, I think it's fine.

Also, i suppose as long as we're unit testing the the data models that create the JSON in the first place, we're pretty safe.

Reply:
This is just so we don't have to re-serialize the json blob back into a RockyRequest, I figured it was one less step to worry about.

TBH, I wasn't thinking about testing issues when I did this, more so keeping the code simple.

That said, and thinking on this more, we should be able to serialize the entire RockyRequest right into Room rather than making it a JSON blob. This should solve everything for both simplicity and testing.

Reply:
> we should be able to serialize the entire RockyRequest right into Room

This is a bit more complex than I thought. I'm going to just use an adapter to deserialize everything back into a RockyRequest before uploading.